packages/vue-components/src/CollapseExpandButtons.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="button-container">
+    <div
+      class="expand-all-button"
+      @click="expandAll()"
+    >
+      <i :class="['far fa-plus-square']"></i>
```

Comment:
I think you don't need :class here because its static not dynamic :) Same  for below!